src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -7607,6 +7607,99 @@ void Method()
 
     void Goo(ref bool b) { }
     void Goo(int i) { }
+}");
+        }
+
+        [WorkItem(26993, "https://github.com/dotnet/roslyn/issues/26993")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateVariable)]
+        public async Task TestGenerateFieldInExpressionBodiedGetter()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Program
+{
+    public int Y
+    {
+        get => [|y|];
```

Comment:
could you add a test with a getter with different accessibility?  Thanks!

Reply:
I added a test where the getter has a different accessibility.